mrp_multi_level/models/product_mrp_area.py

```diff
@@ -7,12 +7,14 @@
 from math import ceil
 
 from odoo import api, fields, models, _
+from odoo.osv import expression
 from odoo.exceptions import ValidationError
 
 
 class ProductMRPArea(models.Model):
     _name = 'product.mrp.area'
     _description = 'Product MRP Area'
+    _rec_name = "product_id"
```

Comment:
A name_get method is already in place. what's the benefit of this?

Reply:
https://github.com/OCA/manufacture/pull/446/files#diff-9ca74319d3d61b1d491d69f9c387dba0R131